### DISCUSSION TITLE
softbody simulation

### DISCUSSION THREAD

**Discussion by nnWhisperer** [google/brax#432] [General]
Hello, I'd like to make simulations of softbodies. Brax is suitable for rigidbody simulation.
As far as I see, Mujoco supports soft bodies too. [The link says "...to model ropes, cloth, and deformable 3D objects."](https://mujoco.readthedocs.io/en/stable/overview.html)
So, as far as I understood in #409 , in the future when mjx becomes the default pipeline, Brax will be able to work even with softbodies, right?
Thank you

**Comment by btaba** [comment]
Hi @nnWhisperer , to see a list of features slated for MuJoCo-MJX and parity with MuJoCo, see [here](https://github.com/google/brax/blob/16304037a36b1d9c8c0b3084f57d1159627b636b/brax/training/acting.py#L34-L55). They are not 1:1 with MuJoCo; I don't expect soft-bodies in MJX/brax to be supported soon
